pathfinder/paizo/player_companion/blood_of_the_elements/bote_spells.lst

```diff
@@ -0,0 +1,9 @@
+# CVS $Revision: $ $Author: $ -- Fri Jan 29 20:50:52 2016 -- reformated by PCGen PrettyLST v6.06.00
+SOURCELONG:Blood of the Elements	SOURCESHORT:BOTE	SOURCEWEB:http://paizo.com/products/btpy978p?Pathfinder-Player-Companion-Blood-of-the-Elements	SOURCEDATE:2014-06
+# ORIGINAL ENTRY: Gwen Thebich (evilpixie87)
+# ORIGINAL ENTRY DATE: 4 April 2016
+
+#Verify: Storm step
+
+Enshroud Thoughts				TYPE:Arcane.Divine	CLASSES:Alchemist,Bard,Inquisitor,Witch=2|Ranger=3		SCHOOL:Enchantment 									DESCRIPTOR:Mind-affecting			COMPS:V, S											CASTTIME:1 standard action		RANGE:Personal		TARGETAREA:You							DURATION:(10*CASTERLEVEL) minutes [D]		SAVEINFO:Will negates (harmless)		SPELLRES:Yes (harmless)										SOURCEPAGE:p.13				DESC:You become warded against the mental prying of others, including the effects of divination spells such as detect thoughts and seek thoughts as well as enchantment spells like modify memory and memory lapse. If another creature attempts to target you with a mind-affecting spell that detects or alters your thoughts or memories, the caster must succeed at a caster level check against a DC of 11 + your caster level or the spell fails. Only spells that detect or alter your thoughts or memories are blocked by this spell; effects such as locate creature, detect evil, and clairaudience/clairvoyance continue to affect you as normal.
+Storm Step						TYPE:Arcane.Divine	CLASSES:Magus,Sorcerer,Wizard,Witch=3					SCHOOL:Conjuration		SUBSCHOOL:Teleportation		DESCRIPTOR:Electricity				COMPS:V, S											CASTTIME:1 standard action		RANGE:Close			TARGETAREA:You							DURATION:Instantaneous						SAVEINFO:Reflex halves					SPELLRES:Yes												SOURCEPAGE:p.13				DESC:You are able to harness the power of the storm to transport yourself a short distance, by taking the form of a furious, sizzling bolt of elemental electricity. You must either have line of sight to your destination or you must specify a direction and distance within range. Creatures and objects in the path of your passage take 1%d8 points of electricity damage. A successful Reflex save halves the damage. If your path intersects with a solid object, you damage the barrier accordingly. If the damage is enough to break through the barrier, you continue beyond the barrier as long as the spell�s range permits; otherwise, your movement stops in the square adjacent to the barrier and the effect ends.|min(5,floor(CASTERLEVEL/2))
```

Comment:
Spells and DESC work slightly different, this:
`DESC:You are able to harness the power of the storm to transport yourself a short distance, by taking the form of a furious, sizzling bolt of elemental electricity. You must either have line of sight to your destination or you must specify a direction and distance within range. Creatures and objects in the path of your passage take 1%d8 points of electricity damage. A successful Reflex save halves the damage. If your path intersects with a solid object, you damage the barrier accordingly. If the damage is enough to break through the barrier, you continue beyond the barrier as long as the spell�s range permits; otherwise, your movement stops in the square adjacent to the barrier and the effect ends.|min(5,floor(CASTERLEVEL/2))`

Should be:
`DESC:You are able to harness the power of the storm to transport yourself a short distance, by taking the form of a furious, sizzling bolt of elemental electricity. You must either have line of sight to your destination or you must specify a direction and distance within range. Creatures and objects in the path of your passage take (CASTERLEVEL/2,5)d8 points of electricity damage. A successful Reflex save halves the damage. If your path intersects with a solid object, you damage the barrier accordingly. If the damage is enough to break through the barrier, you continue beyond the barrier as long as the spell's range permits; otherwise, your movement stops in the square adjacent to the barrier and the effect ends.`

In anywhere other file, the format should be "%1d8" not 1%d8" the "%" always comes first followed by the order they appear in starting at 1